scripts/Muon/GUI/Common/contexts/context_setup.py

```diff
@@ -12,5 +13,19 @@ def setup_context_for_tests(parent_object):
     parent_object.data_context = MuonDataContext(parent_object.loaded_data)
     parent_object.gui_context = MuonGuiContext()
     parent_object.group_context = MuonGroupPairContext(parent_object.data_context.check_group_contains_valid_detectors)
+    parent_object.phase_table_context = PhaseTableContext()
     parent_object.context = MuonContext(muon_data_context=parent_object.data_context, muon_group_context=parent_object.group_context,
-                                        muon_gui_context=parent_object.gui_context)
+                                        muon_gui_context=parent_object.gui_context, muon_phase_context=parent_object.phase_table_context)
+
+
+def setup_context():
+    loaded_data = MuonLoadData()
+    loaded_data.get_main_field_direction = mock.MagicMock(return_value='transverse')
```

Comment:
why do we have a mock in the context?

Reply:
This is a test helper function I'll move it to tests so this is clearer